gymnasium_robotics/envs/maze/ant_maze.py

```diff
@@ -271,21 +272,27 @@ def reset(self, *, seed: Optional[int] = None, **kwargs):
 
         obs, info = self.ant_env.reset(seed=seed)
         obs_dict = self._get_obs(obs)
+        info["success"] = bool(
+            np.linalg.norm(obs_dict["achieved_goal"] - self.goal) <= 0.45
+        )
```

Comment:
why is this here?

Reply:
`info` should be generated for reset as well. Although, this should always be `False` it's good to compute it just to double check. We should include this in the tests.

Reply:
Good idea, will add a test.